src/main/java/evolution/entities/Continent.java

```diff
@@ -16,20 +16,24 @@
     private final int DEFENDER_WIN = 1;
     private final int ATTACKER_WIN = 2;
     private final int DIFFERENCE_BETWEEN_PROPERTIES = 5;
+    private final int NUMBER_OF_ART_WORKS_FOR_GOLDEN_AGE = 5;
```

Comment:
artworks זו מילה אחת 
או artworks או  Works of art

Reply:
תוקן